actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -454,7 +454,7 @@ def initialize(parent_jar, key_generator, options = {})
         @parent_jar = parent_jar
         @options = options
         secret = key_generator.generate_key(@options[:signed_cookie_salt])
-        @verifier = ActiveSupport::MessageVerifier.new(secret, digest: digest, serializer: ActiveSupport::MessageEncryptor::NullSerializer)
+        @verifier = ActiveSupport::MessageVerifier.new(secret, digest: digest, serializer: serializer)
```

Comment:
Let's just focus on making the verifier spit out JWTs and leave cookies.rb untouched for now.